src/main/java/kitchenpos/products/tobe/application/TobeProductService.java

```diff
@@ -0,0 +1,70 @@
+package kitchenpos.products.tobe.application;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+import kitchenpos.menus.domain.Menu;
+import kitchenpos.menus.domain.MenuProduct;
+import kitchenpos.menus.domain.MenuRepository;
+import kitchenpos.products.tobe.domain.Product;
+import kitchenpos.products.tobe.domain.ProductId;
+import kitchenpos.products.tobe.domain.TobeProductRepository;
+import kitchenpos.shared.client.PurgomalumClient;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class TobeProductService {
+
+    private final TobeProductRepository productRepository;
+    private final MenuRepository menuRepository;
+    private final PurgomalumClient purgomalumClient;
+
+    public TobeProductService(
+        final TobeProductRepository productRepository,
+        final MenuRepository menuRepository,
+        final PurgomalumClient purgomalumClient
+    ) {
+        this.productRepository = productRepository;
+        this.menuRepository = menuRepository;
+        this.purgomalumClient = purgomalumClient;
+    }
+
+    @Transactional
+    public Product create(final Product request) {
+        final BigDecimal price = request.getPrice();
+        final String name = request.getName();
+        final UUID id = UUID.randomUUID();
+        final Product product = new Product(id, name, purgomalumClient, price);
+        return productRepository.save(product);
+    }
+
+    @Transactional
+    public Product changePrice(final UUID productId, final Product request) {
+        final Product product = productRepository.findById(new ProductId(productId))
+            .orElseThrow(NoSuchElementException::new);
```

Comment:
다음과 같은 방식으로 예외에 메세지를 추가한다면 디버깅이 더 쉬워질 것 같아요. 😄 
 ```suggestion
             .orElseThrow(() -> new NoSuchElementException("Product not found with id: " + productId));
```

Reply:
디버깅에 유용할 것 같아요!